contributors/devel/sig-architecture/api-conventions.md

```diff
@@ -5,7 +5,7 @@ Updated: 3/7/2017
 
 *This document is oriented at users who want a deeper understanding of the
 Kubernetes API structure, and developers wanting to extend the Kubernetes API.
-An introduction to using resources with kubectl can be found in [the object management overview](https://kubernetes.io/docs/tutorials/object-management-kubectl/object-management/).*
+An introduction to using resources with kubectl can be found in [the declarative object management overview](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/declarative-config/).*
```

Comment:
Sorry for the back and forth, but I think this should actually be https://kubernetes.io/docs/concepts/overview/working-with-objects/object-management/

This is the current version of the document that https://v1-10.docs.kubernetes.io/docs/tutorials/object-management-kubectl/object-management/ goes to.

```suggestion
An introduction to using resources with kubectl can be found in [the object management overview](https://kubernetes.io/docs/concepts/overview/working-with-objects/object-management/).*
```